test/core/security/grpc_tls_crl_provider_test.cc

```diff
@@ -0,0 +1,100 @@
+//
+//
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+#include <grpc/support/port_platform.h>
+
+#include "src/core/lib/security/credentials/tls/grpc_tls_crl_provider.h"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "absl/status/statusor.h"
+
+#include <grpc/grpc_audit_logging.h>
+#include <grpc/grpc_crl_provider.h>
+
+#include "test/core/util/test_config.h"
+#include "test/core/util/tls_utils.h"
+
+#define CRL_PATH "test/core/tsi/test_creds/crl_data/crls/ab06acdd.r0"
```

Comment:
Instead of using macros, please use constants.

```
constexpr char* kCrlPath = "test/core/tsi/test_creds/crl_data/crls/ab06acdd.r0";
constexpr absl::string_view kCrlIssuer = "/C=AU/ST=Some-State/O=Internet Widgits Pty Ltd/CN=testca";
```

Reply:
Changed them to const char* - `constexpr` was giving me `ISO C++11 does not allow conversion from string literal to 'char *const'clang(-Wwritable-strings)`